src/app/footer/footer.component.ts

```diff
@@ -1,10 +1,11 @@
 import { Component } from '@angular/core';
+import { APP_ROUTES } from '../shared/globals/routes';
 
 @Component({
   selector: 'app-footer',
   templateUrl: './footer.component.html',
   styleUrls: ['./footer.component.scss']
 })
 export class FooterComponent {
-
+  public APP_ROUTES = APP_ROUTES;
```

Comment:
if you are directly using the constant APP_ROUTES, you don't need to initialize it inside another property